Vagrantfile

```diff
@@ -29,6 +29,7 @@ Vagrant::configure("2") do |config|
   end
 
   config.vm.define "clean" do |vm|
```

Comment:
Hm I think it would be useful to keep it around, but make the clean the default would be good idea!